01/most-frequent-k-elements.md

```diff
@@ -201,14 +160,10 @@ Write the logical steps here.
 
 ##### !hint
 1. Handle edge cases
-2. Initialize list to hold triangle
-3. Loop from 0 to num_rows
-    1. Initialize list for current row by setting the size to i + 1
-    2. Set the first and last values of the row to 1
-    3. Loop through the rest of the row
-        1. Calculate the current element in the row by adding the elements in the row above and to the left and above and to the right
-    4. Append the row list to the triangle list
-4. Return the triangle list
+2. Initialize hash that holds integer occurences
+3. Loop through unique integers
+    1. Sort them in descending order
+4. Return list
```

Comment:
Could you give a little more detail as to what you mean by this and what you think should be here instead?

Reply:
In the classroom material, the sample approaches were listed as an explanation rather than a hint. Hints are available from the get go, while explanations aren't available until an answer is submitted. I wouldn't expect a "hint" to be the actual answer. If I were a student in this situation, I'd be disappointed if I only wanted a hint but was given the actual answer.

Reply:
Oh okay okay, gotcha gotcha! I thought you meant like that section was suppose to be an explanation but I just need to not give them the answer!

Reply:
Oh yeah. Sorry. I just meant to use
```
##### !explanation 
##### !end-explanation 
```

rather than 
```
##### !hint
##### !end-hint 
```